doc-site/docs/tutorials/custom_contracts/cardano.md

```diff
@@ -0,0 +1,542 @@
+---
+title: Cardano
+---
+
+# Work with Cardano dApps
+
+This guide describes the steps to author and deploy a Cardano dApp through FireFly.
+
+## What is a Cardano dApp?
+
+Cardano dApps typically have two components: off-chain and on-chain.
+
+ - The off-chain component is written in an ordinary programming language using a Cardano-specific library. It builds transactions, signs them with a private key, and submits them to be published to the chain. The FireFly Cardano connector uses a framework called [Balius](https://github.com/txpipe/balius) for off-chain code. This lets you write transaction-building logic in Rust, which is compiled to WebAssembly and run in response to HTTP requests or new blocks reaching the chain.
+
+ - The on-chain component is a "validator script". Validator scripts are written in domain-specific languages such as [Aiken](https://aiken-lang.org/), and compiled to a bytecode called [UPLC](https://aiken-lang.org/uplc). They take a transaction as input, and return true or false to indicate whether that transaction is valid. ADA and native takens can be locked at the address of one of these scripts; if they are, then they can only be spent by transactions which the script considers valid.
+
+## Writing a dApp
+
+First, decide on the contract which your dApp will satisfy. FireFly uses [JSON schema](https://json-schema.org/) to describe its contracts. Create a file named `contract.json`. An example is below:
```

Comment:
More than JSONSchema it uses FireFly Interface Format, can you link to that page please :)